src/Akeneo/Platform/Bundle/UIBundle/Resources/config/requirejs.yml

```
@@ -1245,3 +1245,7 @@ config:
     pim/template/catalog-volume/section:      pimui/templates/catalog-volume/section.html
     pim/template/catalog-volume/number:       pimui/templates/catalog-volume/number.html
     pim/template/catalog-volume/average-max:     pimui/templates/catalog-volume/average-max.html
+
+
+    pim/edition: pimui/js/pim-edition.ts
+    pim/common/edit/field/file-path: pimui/js/job/common/edit/field/file-path.ts
```

```suggestion
    pim/job/common/edit/field/file-path: pimui/js/job/common/edit/field/file-path.ts
```

to be consistent with others modules?